### ISSUE TITLE
Building frontend assets fails

### ISSUE THREAD

**Issue by sheppe** [grafana/grafana#15927]
**What happened**:
I updated the code with the fix for https://github.com/grafana/grafana/issues/15922 and now running "yarn start" leads to the following error:
[SASS variables generator] C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:226
[SASS variables generator]     return new TSError(diagnosticText, diagnosticCodes)
[SASS variables generator]            ^
[SASS variables generator] TSError: ⨯ Unable to compile TypeScript:
[SASS variables generator] scripts/cli/generateSassVariableFiles.ts(2,23): error TS2307: Cannot find module '@grafana/ui/src/themes/dark'.
[SASS variables generator] scripts/cli/generateSassVariableFiles.ts(3,24): error TS2307: Cannot find module '@grafana/ui/src/themes/light'.
[SASS variables generator] scripts/cli/generateSassVariableFiles.ts(4,26): error TS2307: Cannot find module '@grafana/ui/src/themes/default'.
[SASS variables generator] scripts/cli/generateSassVariableFiles.ts(5,39): error TS2307: Cannot find module '@grafana/ui/src/themes/_variables.dark.scss.tmpl'.
[SASS variables generator] scripts/cli/generateSassVariableFiles.ts(6,40): error TS2307: Cannot find module '@grafana/ui/src/themes/_variables.light.scss.tmpl'.
[SASS variables generator] scripts/cli/generateSassVariableFiles.ts(7,41): error TS2307: Cannot find module '@grafana/ui/src/themes/_variables.scss.tmpl'.
[SASS variables generator]
[SASS variables generator]     at createTSError (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:226:12)
[SASS variables generator]     at getOutput (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:335:40)
[SASS variables generator]     at Object.compile (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:368:11)
[SASS variables generator]     at Module.m._compile (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:414:43)
[SASS variables generator]     at Module._extensions..js (internal/modules/cjs/loader.js:712:10)
[SASS variables generator]     at Object.require.extensions.(anonymous function) [as .ts] (C:\xyz\go\src\github.com\grafana\grafana\node_modules\ts-node\src\index.ts:417:12)
[SASS variables generator]     at Module.load (internal/modules/cjs/loader.js:600:32)
[SASS variables generator]     at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
[SASS variables generator]     at Function.Module._load (internal/modules/cjs/loader.js:531:3)
[SASS variables generator]     at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.

**What you expected to happen**: 
The frontend assets to build.

**How to reproduce it (as minimally and precisely as possible)**:
Same as the linked issue: follow the steps at your "building from source page" down to running "yarn start"

**Anything else we need to know?**:
I deleted the node_modules directory and reran "npm install --pure-lockfile", but it didn't fix anything.

**Environment**:
- Grafana version: the one I downloaded from your site today.
- Data source type & version:
- OS Grafana is installed on: Windows 10 Enterprise
- User OS & Browser: Windows 10 Enterprise
- Grafana plugins: None
- Others:

NPM version: 6.4.1
Node version: 10.15.3
Go version: 1.11
gcc version: 5.1.0

**Comment by sheppe** [comment]
This error ends up with the batch job spitting out and hanging on the following:

[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/datasources/state/navModel.ts(2,48):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/datasources/state/reducers.test.ts(18,32):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/datasources/state/reducers.ts(2,36):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/datasources/state/selectors.ts(1,36):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/Explore.tsx(24,97):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/ExploreToolbar.tsx(6,63):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/Graph.tsx(10,30):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/GraphContainer.tsx(4,41):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/Logs.tsx(5,38):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/LogsContainer.tsx(4,41):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/QueryEditor.tsx(10,41):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/QueryRow.tsx(16,90):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/state/actions.ts(34,8):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/state/actionTypes.ts(10,8):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/state/reducers.ts(9,27):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/TimePicker.tsx(6,41):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/explore/Wrapper.tsx(12,33):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/folders/FolderPermissions.tsx(5,25):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/plugins/datasource_srv.ts(10,53):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/plugins/plugin_loader.ts(21,31):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/plugins/plugin_loader.ts(29,28):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/plugins/state/navModel.ts(9,48):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/teams/TeamGroupSync.tsx(4,25):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/teams/TeamList.tsx(5,30):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/teams/TeamMembers.tsx(5,30):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/teams/TeamSettings.tsx(3,27):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/templating/datasource_variable.ts(2,33):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/templating/query_variable.ts(3,33):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/templating/template_srv.ts(4,39):
[Webpack] TS2307: Cannot find module '@grafana/ui/src'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/features/templating/variable_srv.ts(14,27):
[Webpack] TS2307: Cannot find module '@grafana/ui/src'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/graphite/FunctionEditor.tsx(2,42):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryEditor.tsx(8,34):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(20,62):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(88,15):
[Webpack] TS2339: Property 'datasource' does not exist on type 'LokiQueryFieldProps'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(89,37):
[Webpack] TS2339: Property 'datasource' does not exist on type 'LokiQueryFieldProps'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(94,37):
[Webpack] TS2339: Property 'onExecuteQuery' does not exist on type 'LokiQueryFieldProps'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(101,57):
[Webpack] TS2339: Property 'onExecuteQuery' does not exist on type 'LokiQueryFieldProps'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(165,13):
[Webpack] TS2339: Property 'query' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(165,20):
[Webpack] TS2339: Property 'onQueryChange' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(165,35):
[Webpack] TS2339: Property 'onExecuteQuery' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(177,13):
[Webpack] TS2339: Property 'hint' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(177,19):
[Webpack] TS2339: Property 'onExecuteHint' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(217,13):
[Webpack] TS2339: Property 'error' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(217,20):
[Webpack] TS2339: Property 'hint' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(217,26):
[Webpack] TS2339: Property 'query' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiQueryField.tsx(241,42):
[Webpack] TS2339: Property 'onExecuteQuery' does not exist on type 'Readonly<LokiQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/components/LokiStartPage.tsx(3,39):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/datasource.ts(14,46):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/language_provider.test.ts(97,52):
[Webpack] TS2322: Type '{ refId: string; expr: string; }' is not assignable to type 'LokiQuery'.
[Webpack]   Object literal may only specify known properties, and 'refId' does not exist in type 'LokiQuery'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/language_provider.ts(178,20):
[Webpack] TS2339: Property 'refId' does not exist on type 'LokiQuery'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/loki/types.ts(1,27):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(16,62):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(109,15):
[Webpack] TS2339: Property 'datasource' does not exist on type 'PromQueryFieldProps'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(110,37):
[Webpack] TS2339: Property 'datasource' does not exist on type 'PromQueryFieldProps'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(115,37):
[Webpack] TS2339: Property 'onExecuteQuery' does not exist on type 'PromQueryFieldProps'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(173,13):
[Webpack] TS2339: Property 'query' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(173,20):
[Webpack] TS2339: Property 'onQueryChange' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(173,35):
[Webpack] TS2339: Property 'onExecuteQuery' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(185,13):
[Webpack] TS2339: Property 'hint' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(185,19):
[Webpack] TS2339: Property 'onExecuteHint' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(242,13):
[Webpack] TS2339: Property 'error' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(242,20):
[Webpack] TS2339: Property 'hint' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(242,26):
[Webpack] TS2339: Property 'query' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromQueryField.tsx(265,42):
[Webpack] TS2339: Property 'onExecuteQuery' does not exist on type 'Readonly<PromQueryFieldProps> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/components/PromStart.tsx(3,39):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/datasource.ts(18,49):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/query_hints.ts(2,27):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/prometheus/types.ts(1,27):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/stackdriver/components/Alignments.tsx(6,34):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/stackdriver/components/QueryEditor.tsx(16,34):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/stackdriver/datasource.ts(6,49):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/stackdriver/types.ts(1,27):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/testdata/datasource.ts(3,49):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/testdata/QueryEditor.tsx(9,53):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/testdata/QueryEditor.tsx(12,34):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/datasource/testdata/types.ts(1,27):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugeOptionsBox.tsx(5,43):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugeOptionsBox.tsx(8,45):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(5,49):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(8,23):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(12,60):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(28,20):
[Webpack] TS2339: Property 'panelData' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(28,44):
[Webpack] TS2339: Property 'panelData' does not exist on type 'Props'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(34,13):
[Webpack] TS2339: Property 'panelData' does not exist on type 'Props'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(34,24):
[Webpack] TS2339: Property 'options' does not exist on type 'Props'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(53,13):
[Webpack] TS2339: Property 'width' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(53,20):
[Webpack] TS2339: Property 'height' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(53,28):
[Webpack] TS2339: Property 'replaceVariables' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanel.tsx(53,46):
[Webpack] TS2339: Property 'options' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/GaugePanelEditor.tsx(9,8):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/module.tsx(1,34):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/SingleStatValueEditor.tsx(5,77):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/gauge/types.ts(1,41):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph/axes_editor.ts(1,33):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph/data_processor.ts(2,50):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph/graph.ts(28,32):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph/Legend/Legend.tsx(4,33):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph/Legend/LegendSeriesItem.tsx(4,35):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph/module.ts(13,42):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph/threshold_manager.ts(4,42):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph/time_region_manager.ts(4,60):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanel.tsx(6,35):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/utils'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanel.tsx(9,23):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanel.tsx(12,58):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanel.tsx(19,13):
[Webpack] TS2339: Property 'panelData' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanel.tsx(19,24):
[Webpack] TS2339: Property 'timeRange' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanel.tsx(19,35):
[Webpack] TS2339: Property 'width' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanel.tsx(19,42):
[Webpack] TS2339: Property 'height' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanel.tsx(20,60):
[Webpack] TS2339: Property 'options' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/GraphPanelEditor.tsx(6,42):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/graph2/module.tsx(1,34):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/heatmap/color_legend.ts(8,60):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/heatmap/heatmap_tooltip.ts(5,32):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/heatmap/rendering.ts(10,76):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/singlestat/module.ts(11,76):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/table/column_options.ts(2,33):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/table/renderer.ts(3,93):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/table/specs/renderer.test.ts(4,42):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/text2/module.tsx(1,34):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/text2/TextPanel.tsx(12,28):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/text2/TextPanel.tsx(31,49):
[Webpack] TS2339: Property 'options' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/text2/TextPanel.tsx(44,13):
[Webpack] TS2339: Property 'replaceVariables' does not exist on type 'Readonly<Props> & Readonly<{ children?: ReactNode; }>'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/plugins/panel/text2/TextPanelEditor.tsx(5,79):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/routes/GrafanaCtrl.ts(7,24):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/types/datasources.ts(3,36):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/types/explore.ts(11,8):
[Webpack] TS2307: Cannot find module '@grafana/ui'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/app/types/plugins.ts(1,47):
[Webpack] TS2307: Cannot find module '@grafana/ui/src/types'.
[Webpack] ERROR in C:/xyz/go/src/github.com/grafana/grafana/public/test/helpers/getQueryOptions.ts(1,45):
[Webpack] TS2307: Cannot find module '@grafana/ui'.

**Comment by torkelo** [comment]
Make sure you have node lts installed and run yarn install (any errors from running yarn install?)

**Comment by sheppe** [comment]
Like I mention at the bottom of the issue, I have Node LTS installed. And like I mention in the issue I linked to, all other commands on the page up to that point finish successfully. I can even start the Grafana backend, but it has no assets to serve.

**Comment by sheppe** [comment]
> 
> 
> Make sure you have node lts installed and run yarn install (any errors from running yarn install?)

Like I mention at the bottom of the issue, I have Node LTS installed. And like I mention in the issue I linked to, all other commands on the page up to that point finish successfully. I can even start the Grafana backend, but it has no assets to serve.

I don't know why you closed this issue before hearing back from me. The last issue I reported yesterday uncovered a bug.
